src/main/java/core/basesyntax/ColorSupplire.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplire {
+    private Random random = new Random();

```

Comment:
Consider making the `Random` object `final` since it's not expected to change during the lifetime of the `ColorSupplier` object.